extensions/VisualEditor/modules/ve-mw/init/ve.init.mw.Target.js

```diff
@@ -293,6 +293,28 @@ ve.init.mw.Target.static.integrationType = 'page';
 /* Static Methods */
 
 /**
+ * Process page query arguments
+ * @returns {object} The processed parameters
+ */
+function processArguments() {
```

Comment:
No need for this whole method - just look how other parameters are accessed - for instance "veaction": ```ve.init.wikia.ViewPageTarget.init.js``` - line 396